metadata.json

```diff
@@ -11,7 +11,7 @@
     },
     {
       "name": "puppetlabs/yumrepo_core",
-      "version_requirement": ">=1.0.0 <2.0.0"
+      "version_requirement": ">=1.0.0 < 3.0.0"
```

Comment:
IMO the *_core modules shouldn't be listed here. They are supposed to be always present on the systems. I suggest to remove it here.

Reply:
would that be a backwards-incompatible change?

Reply:
I don't think so. I would call it a bugfix. The module in the metadata.json doesn't bring any value.